com/truckmuncher/api/trucks.proto

```diff
@@ -30,11 +30,20 @@ service TruckService {
     /**
      * Modify the serving mode for a truck.
      *
-     * This call requires Vendor authorization
+     * This call is an alias for reportServingMode
+     * This call is deprecated and should no longer be used by clients. It will be removed in the near future.
+     * This call requires the user to be logged in
      */
     rpc modifyServingMode (ServingModeRequest) returns (ServingModeResponse);
```

Comment:
Right now we dont have a concept of reporting locations and we use the modifyServingMode method. Trucks use this to put themselves on the map. Im saying are trucks going to continue to use that for that purpose? and then this reportServingMode be only for crowdsourcing? or will they be combined?